staging/src/k8s.io/client-go/tools/cache/store.go

```diff
@@ -59,6 +59,7 @@ type Store interface {
 
 	// GetByKey returns the accumulator associated with the given key
 	GetByKey(key string) (item interface{}, exists bool, err error)
+	Count() int64
```

Comment:
nit:
1. Here is a through, not have to follow, feel free to ignore this comment if I'm wrong. Do we have to change the underlay interface in `client-go`? It make this PR a really big change. Can we use existed `ListKeys()`? I know this method will do an Iteration and copy all the keys, there is some cost, but it looks simplier and safier.

2. in PR describe, you said:
> return count from watching cache directly, this function only used by metrics

   Strictly speaking, the resource count is not only used by metrics, but also used by APF to estimate request cost. But I think it's fine to use cache to get the count number, since it doesn't have to be very accurate too.

Reply:
Good ieda, I change to use ListKeys, which make commit clearly.  Add Count() function should be in another commit.